appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -95,6 +95,7 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.ERROR_SELECT_VALID_ITEM_FROM_DROPDOWN = "Select a valid item in the drop down.";
     Blockly.ERROR_DUPLICATE_EVENT_HANDLER = "This is a duplicate event handler for this component.";
     Blockly.ERROR_COMPONENT_DOES_NOT_EXIST = "Component does not exist";
+    Blockly.ERROR_BLOCK_IS_NOT_DEFINED = "This block is not defined. Delete this block!";
```

Comment:
Don't we need to translate this text for other version? 
I could only modify for msg/en/ ..